### PR TITLE
Manta: add linux-aarch64 builds

### PR DESCRIPTION
**Pull request by dslarm**
Adds linux-aarch64 builds by using source instead of binary.

### CODE REVIEW

**Comment by dslarm** [comment]
@BiocondaBot please add label

File: recipes/manta/meta.yaml [hunk 1]
```diff
@@ -6,17 +6,28 @@ package:
...
   run_exports:
     - {{ pin_subpackage(name, max_pin="x") }}
+  skip: True  # [osx] # until CI os/x is fixed https://github.com/bioconda/bioconda-recipes/issues/52553
 
 requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - boost-cpp
```
Comment (martin-g) [inline recipes/manta/meta.yaml:25]: this should be in `host`

File: recipes/manta/meta.yaml [hunk 2]
```diff
@@ -6,17 +6,28 @@ package:
...
+    - python ==2.7.*
```
Comment (martin-g) [inline recipes/manta/meta.yaml:26]: This is not needed in `build`. `host` should be enough

File: recipes/manta/meta.yaml (hunk 1 above)
Comment (dslarm) [reply recipes/manta/meta.yaml:27]: That fails to lint:
```
The recipe requests a compiler in a section other than build
Please move the `{{ compiler('language') }}` line into the`requirements: build:` section
```

Comment (martin-g) [reply recipes/manta/meta.yaml]: Sorry, I meant just `boost-cpp`.
`build` should list only build **tools**. All software _libraries_ should be in `host`

Comment (dslarm) [reply recipes/manta/meta.yaml]: Moving boost-cpp seems to be better - just trying that

File: recipes/manta/meta.yaml [hunk 3]
```diff
@@ -6,17 +6,28 @@ package:
...
   run_exports:
     - {{ pin_subpackage(name, max_pin="x") }}
+  skip: True  # [osx] # until CI os/x is fixed https://github.com/bioconda/bioconda-recipes/issues/52553
 
 requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - python ==2.7.*
```
Comment (martin-g) [inline recipes/manta/meta.yaml:25]:
```suggestion
```
Interpreters should be in `host` too

**Review by martin-g — APPROVED** [review]

**Pull request merged**